### PR TITLE
Removed code from signup flow

### PR DESCRIPTION
**Pull request by AndreSand**
Hi Guys, can you guys merge? I don't see any code conflicts. 

I remove code from signup flow.

I'm pushing UI changes/improvements (not new functionalities) @dbykovsky @alberto-campos 

My fork is up to date with repo M.

Thanks,

### CODE REVIEW

File: app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java [hunk 1]
```diff
@@ -105,27 +135,66 @@ private void signup() {
...
-        // Set up a new Parse user
-        ParseUser user = new ParseUser();
-        user.setUsername(username);
-        user.setEmail(uemail);
-        user.setPassword(password);
 
-        // Call the Parse signup method
-        user.signUpInBackground(new SignUpCallback() {
+        //ADD IMAGE Hardcoded for now. Work in Progress
+        Bitmap bitmap = BitmapFactory.decodeResource(getResources(),
```
Comment (dbykovsky) [inline app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java:124]: What is the purpose of this code in  Sign up activity? @alberto-campos

**Comment by AndreSand** [comment]
Fix issues from https://github.com/dbykovsky/sumofusM/issues/53

@dbykovsky @alberto-campos can you guys merge, before there are merge code conflicts?

**Comment by AndreSand** [comment]
Latest push fixes issues from https://github.com/dbykovsky/sumofusM/issues/53

@dbykovsky @alberto-campos can you guys merge, before there are merge code conflicts?
pull request https://github.com/dbykovsky/sumofusM/pull/49

**Comment by AndreSand** [comment]
@dbykovsky @alberto-campos can you guys merge, before there are merge code conflicts? 

I have 10 unmerge commits?? I don't think this is correct, we should all have the same code base...

**Comment by AndreSand** [comment]
Everything working fine. Latest push I fixed a registration issue.

@dbykovsky @alberto-campos 

Please merge so you guys can see all my fixes.

Thanks.

File: app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java (hunk 1 above)
Comment (AndreSand) [inline app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java:124]: It's part of Parse API to create user

**Comment by AndreSand** [comment]
@dbykovsky @alberto-campos 

Hi Alberto, can you merge my pull request first? I see conflicts in my pull request now :(

**Comment by alberto-campos** [comment]
Andres, can you remove the option to take picture from user profile. Also you don't need to commit the files that have no new code or code removed.

**Comment by AndreSand** [comment]
@dbykovsky @alberto-campos

Hi Guys, I believed I fixed the issues :+1: :+1:

**Pull request merged**